api/package.json

```diff
@@ -32,6 +32,7 @@
     "graphql-ws": "^5.11.1",
     "ioredis": "^5.0.5",
     "lodash": "^4.17.21",
+    "openskill": "https://github.com/philihp/openskill.js",
```

Comment:
They added types in [a PR](https://github.com/philihp/openskill.js/pull/240) but have not published it to npm :(